app/models/manageiq/providers/embedded_ansible/crud_common.rb

```diff
@@ -50,6 +50,7 @@ def encrypt_queue_params(params)
     end
 
     def create_in_provider_queue(manager_id, params, auth_user = nil)
+      parent.find(manager_id) # validation that the manager ID is from EmbeddedAnsible
```

Comment:
If we're going to leave this in, then can we disable the `Lint/UselessAssignment` cop?

Reply:
@djberg96 I didn't assign anything, so there is nothing to disable.  That is the one change I did make to this line.